### PR TITLE
Update electron to version 33

### PR DESCRIPTION
**Pull request by aneuwald-ctw**
**User-Facing Changes**
N/A

**Description**
Update Electron version to the latest version 33

Also, Playwright had to be updated, as the versions after Electron v28 have a small incompatibility. An extra step on pipeline was created to install the new browsers needed from the new Playwright version: https://github.com/microsoft/playwright/issues/28048 

Also, it was changed the react-virtualized package to react-virtualized-auto-sizer, as it is an stand alone package resulting in a smaller bundle.

**Checklist**

- [N/A] The web version was tested and it is running ok
- [x] The desktop version was tested and it is running ok
- [N/A] This change is covered by unit tests
- [N/A] Files constants.ts, types.ts and *.style.ts have been checked and relevant code snippets have been relocated

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -109,6 +109,13 @@ jobs:
         if: matrix.os != 'ubuntu-20.04'
         run: yarn run test:integration
 
+      # New version of playwright requires to install new browsers
+      - name: Install Playwright browsers
```
Comment (ctw-joao-luis) [inline .github/workflows/ci.yml:113]: Why was this necessary to be added? Just wanting to understand

**Review by ctw-joao-luis — APPROVED** [review]
Everything looks good to me

**Comment by laisspportugal** [comment]
Just left a comment regarding the info that I though could be usefull on the PR description.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (aneuwald-ctw) [reply .github/workflows/ci.yml:113]: If you check this run on the CI Pipeline, the Playwright itself indicates to install new browsers in order to run correctly:
https://github.com/Lichtblick-Suite/lichtblick/actions/runs/12296062820/job/34314281675

Maybe this can be eliminated in the future, but by now it was the way I found to keep it working smoothly.

File: packages/suite-base/package.json [hunk 2]
```diff
@@ -82,7 +82,7 @@
     "@types/react-dom": "18.3.0",
     "@types/react-hover-observer": "workspace:*",
     "@types/react-transition-group": "^4.4.8",
-    "@types/react-virtualized": "9.21.29",
+    "@types/react-virtualized-auto-sizer": "1.0.4",
```
[the remark this answers is not in the document]
Comment (aneuwald-ctw) [reply packages/suite-base/package.json:85]: It was not exactly replaced, It is the same library but isolated. It means it will load only the AutoSizer to the project instead of all react-virtualized and only use this component. You can check more info directly in the NPM:

https://www.npmjs.com/package/react-virtualized-auto-sizer
